.github/workflows/ci.yml

```
@@ -1,12 +1,12 @@
-name: Ruby
+name: CI
 
-on: [push, pull_request]
+on: [push]
 
 jobs:
   build:
 
     runs-on: ubuntu-latest
-    name: Ruby ${{ matrix.ruby }}
+    name: Ruby ${{ matrix.ruby }} | Gemfile ${{ matrix.gemfile }}
```

This adds a whole lot of duplication with `Gemfile [gG]emfile` at the begining of each label. Maybe: 
```suggestion
    name: Ruby ${{ matrix.ruby }}: ${{ matrix.gemfile }}
```

---

Yeah, that works. I just did the same thing as `Job Iteration`, but it [uses a matrix over the filename basenames](https://github.com/Shopify/job-iteration/blob/6d6af27b0bae5bd6742bff2f5c1473be8be86559/.github/workflows/ci.yml#L17):

```yml
gemfile: [rails_5_2, rails_6_0, rails_edge]
```

Given our setup here, dropping the prefix makes sense.